src/systems/sensors/Sensors.cc

```diff
@@ -599,6 +604,23 @@ void Sensors::Update(const UpdateInfo &_info,
 {
   GZ_PROFILE("Sensors::Update");
   std::unique_lock<std::mutex> lock(this->dataPtr->renderMutex);
+
+#ifdef __APPLE__
+  // On macOS the render engine must be initialised on the main thread.
+  if (!this->dataPtr->initialized &&
+      (_ecm.HasComponentType(components::Camera::typeId) ||
```

Comment:
It's missing the new `BoundingBoxCamera`

Reply:
You're right! Updated in [6ae22bf](https://github.com/gazebosim/gz-sim/pull/1225/commits/6ae22bfac7eda5fa56ab193722eadf3576a93ade) and working fine:

![boundingbox_camera](https://user-images.githubusercontent.com/24916364/187021428-d402a79f-844d-4452-b3d0-16b8a8706d32.png)